scripts/cy-wrapper.sh

```diff
@@ -0,0 +1,64 @@
+#!/usr/bin/env bash
+
+# Warning: this script is expected to be used in cycloid-toolkit docker image.
+# We do not recommand to use it in another context.
+
+if test -z "$BASH_VERSION"; then
+  echo "Please run this script using bash, not sh or any other shell." >&2
+  exit 1
+fi
+
+export CY_API_URL="${CY_API_URL:-https://http-api.cycloid.io}"
+export CY_BINARIES_PATH="${CY_BINARIES_PATH:-/usr/local/bin}"
+
+# Ensure we have Cycloid directory created
+if ! [ -d "$CY_BINARIES_PATH" ]; then
+  mkdir -p "$CY_BINARIES_PATH"
+fi
+
+# If specified in the commandline use it
+if [[ $(echo "$@" | grep -E --  "--api-url(=|\s+)([^\]+)") ]]; then
+  CY_API_URL=$(echo "$@" | sed -r "s/.*--api-url(=|\s+)([^ ]+).*/\2/")
+fi
+
+# Remove trailing /
+CY_API_URL=${CY_API_URL%/}
+
+# Get Cycloid API version
+export CY_VERSION=$(curl -s "${CY_API_URL}/version" | jq -r .data.version)
+
+if [[ -z "$CY_VERSION" ]]; then
+  echo "Error: Unable to get Cycloid API version on ${CY_API_URL}/version"
+  exit 1
+fi
+
+# Download the binary if not present
+export CY_BINARY="${CY_BINARIES_PATH}/cy-${CY_VERSION}"
+if ! [[ -f "${CY_BINARY}" ]]; then
+
+  CY_URL="https://github.com/cycloidio/cycloid-cli/releases/download/v${CY_VERSION}/cy"
+  wget -q -O "${CY_BINARY}" "$CY_URL"
+  STATUS=$?
+
+  if [ $STATUS != 0 ]; then
+    rm -f "${CY_BINARY}"
+  fi
+
+  # In case of 404, fallback on latest develop version
+  if [ $STATUS == 8 ]; then
+    echo "Warning: Unable to download CLI version ${CY_VERSION}. Fallback to latest develop version"
+    CY_BINARY="${CY_BINARIES_PATH}/cy-latest"
```

Comment:
```suggestion
    CY_VERSION="latest"
```
To avoid redefining `CY_BINARY` ?